src/Interpreters/InterpreterSystemQuery.cpp

```diff
@@ -561,6 +561,14 @@ BlockIO InterpreterSystemQuery::execute()
             getContext()->checkAccess(AccessType::SYSTEM_RELOAD_USERS);
             system_context->getAccessControl().reload(AccessControl::ReloadMode::ALL);
             break;
+        case Type::RELOAD_ASYNCHRONOUS_METRICS:
+        {
+            getContext()->checkAccess(AccessType::SYSTEM_RELOAD_ASYNCHRONOUS_METRICS);
+            auto * asynchronous_metrics = system_context->getAsynchronousMetrics();
+            if (asynchronous_metrics)
+                asynchronous_metrics->update(std::chrono::system_clock::now());
```

Comment:
now I see what Alexey was saying
In server metrics there is `updateHeavyMetricsIfNeeded` which has a different update interval.
They won't be updated with  this call unless the interval has passed since last update.
I think you need to add a new argument `bool force` to `update` and `updateImpl` which will ignore such checks.

Reply:
Right, thanks. Updated.